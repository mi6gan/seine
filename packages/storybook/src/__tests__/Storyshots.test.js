import initStoryshots from '@storybook/addon-storyshots';

initStoryshots({
  configPath: `${__dirname}/../../`,
});
