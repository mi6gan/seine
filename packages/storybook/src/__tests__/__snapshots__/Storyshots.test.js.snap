// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Content Flex 1`] = `
<div
  className="Item-sc-10s0ojn-0 Flex-jm331f-0 kcTaZv"
  direction="row"
  id="grid"
  spacing={32}
  type="grid"
  wrap="nowrap"
>
  <div
    className="Item-sc-10s0ojn-0 Table__Container-u530n6-0 lNMTM"
    id="table"
    type="table"
  >
    <table
      className="Table__StyledTable-u530n6-1 hvaebX"
      scale={1}
    >
      <thead>
        <tr>
          <th
            className="TableCell-oc851c-0 lnUsfb"
          >
            first column
          </th>
          <th
            className="TableCell-oc851c-0 lnUsfb"
          >
            second column
          </th>
          <th
            className="TableCell-oc851c-0 lnUsfb"
          >
            third column
          </th>
        </tr>
      </thead>
      <tbody>
        <tr>
          <td
            className="TableCell-oc851c-0 lnUsfb"
          >
            row 1, column 1
          </td>
          <td
            className="TableCell-oc851c-0 lnUsfb"
          >
            row 1, column 2
          </td>
          <td
            className="TableCell-oc851c-0 lnUsfb"
          >
            row 1, column 3
          </td>
        </tr>
        <tr>
          <td
            className="TableCell-oc851c-0 lnUsfb"
          >
            row 2, column 1
          </td>
          <td
            className="TableCell-oc851c-0 lnUsfb"
          >
            row 2, column 2
          </td>
          <td
            className="TableCell-oc851c-0 lnUsfb"
          >
            row 2, column 3
          </td>
        </tr>
        <tr>
          <td
            className="TableCell-oc851c-0 lnUsfb"
          >
            row 3, column 1
          </td>
          <td
            className="TableCell-oc851c-0 lnUsfb"
          >
            row 3, column 2
          </td>
          <td
            className="TableCell-oc851c-0 lnUsfb"
          >
            row 3, column 3
          </td>
        </tr>
      </tbody>
    </table>
  </div>
  <div
    className="Item-sc-10s0ojn-0 Xwmnj RichText-yekvnw-0 hIhKZq DraftEditor/root DraftEditor/alignLeft RichText-yekvnw-0 hIhKZq DraftEditor/root DraftEditor/alignLeft"
    id="text"
    type="draft"
  >
    <div
      className="DraftEditor-root"
    >
      <div
        className="DraftEditor-editorContainer"
      >
        <div
          aria-activedescendant={null}
          aria-autocomplete={null}
          aria-controls={null}
          aria-expanded={null}
          aria-owns={null}
          className="public-DraftEditor-content"
          contentEditable={false}
          onBeforeInput={[Function]}
          onBlur={[Function]}
          onCompositionEnd={[Function]}
          onCompositionStart={[Function]}
          onCopy={[Function]}
          onCut={[Function]}
          onDragEnd={[Function]}
          onDragEnter={[Function]}
          onDragLeave={[Function]}
          onDragOver={[Function]}
          onDragStart={[Function]}
          onDrop={[Function]}
          onFocus={[Function]}
          onInput={[Function]}
          onKeyDown={[Function]}
          onKeyPress={[Function]}
          onKeyUp={[Function]}
          onMouseUp={[Function]}
          onPaste={[Function]}
          onSelect={[Function]}
          role={null}
          spellCheck={false}
          style={
            Object {
              "WebkitUserSelect": "text",
              "outline": "none",
              "userSelect": "text",
              "whiteSpace": "pre-wrap",
              "wordWrap": "break-word",
            }
          }
          suppressContentEditableWarning={true}
        >
          <div
            data-contents="true"
          >
            <h1
              className="RichText-yekvnw-0 hIhKZq"
              data-block={true}
              data-editor="text"
              data-offset-key="h1-0-0"
            >
              <div
                className="public-DraftStyleDefault-block public-DraftStyleDefault-ltr"
                data-offset-key="h1-0-0"
              >
                <span
                  data-offset-key="h1-0-0"
                  style={Object {}}
                >
                  <span
                    data-text="true"
                  >
                    Header one
                  </span>
                </span>
              </div>
            </h1>
            <h2
              className="RichText-yekvnw-0 hIhKZq"
              data-block={true}
              data-editor="text"
              data-offset-key="h2-0-0"
            >
              <div
                className="public-DraftStyleDefault-block public-DraftStyleDefault-ltr"
                data-offset-key="h2-0-0"
              >
                <span
                  data-offset-key="h2-0-0"
                  style={Object {}}
                >
                  <span
                    data-text="true"
                  >
                    Header two
                  </span>
                </span>
              </div>
            </h2>
            <h3
              className="RichText-yekvnw-0 hIhKZq"
              data-block={true}
              data-editor="text"
              data-offset-key="h3-0-0"
            >
              <div
                className="public-DraftStyleDefault-block public-DraftStyleDefault-ltr"
                data-offset-key="h3-0-0"
              >
                <span
                  data-offset-key="h3-0-0"
                  style={Object {}}
                >
                  <span
                    data-text="true"
                  >
                    Header three
                  </span>
                </span>
              </div>
            </h3>
            <h4
              className="RichText-yekvnw-0 hIhKZq"
              data-block={true}
              data-editor="text"
              data-offset-key="h4-0-0"
            >
              <div
                className="public-DraftStyleDefault-block public-DraftStyleDefault-ltr"
                data-offset-key="h4-0-0"
              >
                <span
                  data-offset-key="h4-0-0"
                  style={Object {}}
                >
                  <span
                    data-text="true"
                  >
                    Header four
                  </span>
                </span>
              </div>
            </h4>
            <h5
              className="RichText-yekvnw-0 hIhKZq"
              data-block={true}
              data-editor="text"
              data-offset-key="h5-0-0"
            >
              <div
                className="public-DraftStyleDefault-block public-DraftStyleDefault-ltr"
                data-offset-key="h5-0-0"
              >
                <span
                  data-offset-key="h5-0-0"
                  style={Object {}}
                >
                  <span
                    data-text="true"
                  >
                    Header five
                  </span>
                </span>
              </div>
            </h5>
            <h6
              className="RichText-yekvnw-0 hIhKZq"
              data-block={true}
              data-editor="text"
              data-offset-key="h6-0-0"
            >
              <div
                className="public-DraftStyleDefault-block public-DraftStyleDefault-ltr"
                data-offset-key="h6-0-0"
              >
                <span
                  data-offset-key="h6-0-0"
                  style={Object {}}
                >
                  <span
                    data-text="true"
                  >
                    Header six
                  </span>
                </span>
              </div>
            </h6>
            <div
              className="RichText-yekvnw-0 hIhKZq"
              data-block={true}
              data-editor="text"
              data-offset-key="unstyled-0-0"
            >
              <div
                className="public-DraftStyleDefault-block public-DraftStyleDefault-ltr"
                data-offset-key="unstyled-0-0"
              >
                <span
                  data-offset-key="unstyled-0-0"
                  style={Object {}}
                >
                  <span
                    data-text="true"
                  >
                    This block is draft.js content.
                  </span>
                </span>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots Content Nested Flex 1`] = `
<div
  className="Item-sc-10s0ojn-0 Flex-jm331f-0 kcTaZv"
  direction="row"
  id="grid"
  spacing={32}
  type="grid"
  wrap="nowrap"
>
  <div
    className="Item-sc-10s0ojn-0 Flex-jm331f-0 fQLCBd"
    direction="column"
    id="b3e6d581-6df8-47ee-b2cf-6cb9119edd78"
    spacing={32}
    type="grid"
    wrap="nowrap"
  >
    <div
      className="Item-sc-10s0ojn-0 Xwmnj RichText-yekvnw-0 hIhKZq DraftEditor/root DraftEditor/alignLeft RichText-yekvnw-0 hIhKZq DraftEditor/root DraftEditor/alignLeft"
      id="27053ca8-1f37-4cf6-bb97-a51847b34dc8"
      type="draft"
    >
      <div
        className="DraftEditor-root"
      >
        <div
          className="DraftEditor-editorContainer"
        >
          <div
            aria-activedescendant={null}
            aria-autocomplete={null}
            aria-controls={null}
            aria-expanded={null}
            aria-owns={null}
            className="public-DraftEditor-content"
            contentEditable={false}
            onBeforeInput={[Function]}
            onBlur={[Function]}
            onCompositionEnd={[Function]}
            onCompositionStart={[Function]}
            onCopy={[Function]}
            onCut={[Function]}
            onDragEnd={[Function]}
            onDragEnter={[Function]}
            onDragLeave={[Function]}
            onDragOver={[Function]}
            onDragStart={[Function]}
            onDrop={[Function]}
            onFocus={[Function]}
            onInput={[Function]}
            onKeyDown={[Function]}
            onKeyPress={[Function]}
            onKeyUp={[Function]}
            onMouseUp={[Function]}
            onPaste={[Function]}
            onSelect={[Function]}
            role={null}
            spellCheck={false}
            style={
              Object {
                "WebkitUserSelect": "text",
                "outline": "none",
                "userSelect": "text",
                "whiteSpace": "pre-wrap",
                "wordWrap": "break-word",
              }
            }
            suppressContentEditableWarning={true}
          >
            <div
              data-contents="true"
            >
              <div
                className="RichText-yekvnw-0 hIhKZq"
                data-block={true}
                data-editor="27053ca8-1f37-4cf6-bb97-a51847b34dc8"
                data-offset-key="bppru-0-0"
              >
                <div
                  className="public-DraftStyleDefault-block public-DraftStyleDefault-ltr"
                  data-offset-key="bppru-0-0"
                >
                  <span
                    data-offset-key="bppru-0-0"
                    style={Object {}}
                  >
                    <span
                      data-text="true"
                    >
                      Rich text
                    </span>
                  </span>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      className="Item-sc-10s0ojn-0 Table__Container-u530n6-0 lNMTM"
      id="table"
      type="table"
    >
      <table
        className="Table__StyledTable-u530n6-1 hvaebX"
        scale={1}
      >
        <thead>
          <tr>
            <th
              className="TableCell-oc851c-0 lnUsfb"
            >
              first column
            </th>
            <th
              className="TableCell-oc851c-0 lnUsfb"
            >
              second column
            </th>
            <th
              className="TableCell-oc851c-0 lnUsfb"
            >
              third column
            </th>
          </tr>
        </thead>
        <tbody>
          <tr>
            <td
              className="TableCell-oc851c-0 lnUsfb"
            >
              row 1, column 1
            </td>
            <td
              className="TableCell-oc851c-0 lnUsfb"
            >
              row 1, column 2
            </td>
            <td
              className="TableCell-oc851c-0 lnUsfb"
            >
              row 1, column 3
            </td>
          </tr>
          <tr>
            <td
              className="TableCell-oc851c-0 lnUsfb"
            >
              row 2, column 1
            </td>
            <td
              className="TableCell-oc851c-0 lnUsfb"
            >
              row 2, column 2
            </td>
            <td
              className="TableCell-oc851c-0 lnUsfb"
            >
              row 2, column 3
            </td>
          </tr>
          <tr>
            <td
              className="TableCell-oc851c-0 lnUsfb"
            >
              row 3, column 1
            </td>
            <td
              className="TableCell-oc851c-0 lnUsfb"
            >
              row 3, column 2
            </td>
            <td
              className="TableCell-oc851c-0 lnUsfb"
            >
              row 3, column 3
            </td>
          </tr>
        </tbody>
      </table>
    </div>
  </div>
  <div
    className="Item-sc-10s0ojn-0 Xwmnj RichText-yekvnw-0 hIhKZq DraftEditor/root DraftEditor/alignLeft RichText-yekvnw-0 hIhKZq DraftEditor/root DraftEditor/alignLeft"
    id="text"
    type="draft"
  >
    <div
      className="DraftEditor-root"
    >
      <div
        className="DraftEditor-editorContainer"
      >
        <div
          aria-activedescendant={null}
          aria-autocomplete={null}
          aria-controls={null}
          aria-expanded={null}
          aria-owns={null}
          className="public-DraftEditor-content"
          contentEditable={false}
          onBeforeInput={[Function]}
          onBlur={[Function]}
          onCompositionEnd={[Function]}
          onCompositionStart={[Function]}
          onCopy={[Function]}
          onCut={[Function]}
          onDragEnd={[Function]}
          onDragEnter={[Function]}
          onDragLeave={[Function]}
          onDragOver={[Function]}
          onDragStart={[Function]}
          onDrop={[Function]}
          onFocus={[Function]}
          onInput={[Function]}
          onKeyDown={[Function]}
          onKeyPress={[Function]}
          onKeyUp={[Function]}
          onMouseUp={[Function]}
          onPaste={[Function]}
          onSelect={[Function]}
          role={null}
          spellCheck={false}
          style={
            Object {
              "WebkitUserSelect": "text",
              "outline": "none",
              "userSelect": "text",
              "whiteSpace": "pre-wrap",
              "wordWrap": "break-word",
            }
          }
          suppressContentEditableWarning={true}
        >
          <div
            data-contents="true"
          >
            <h1
              className="RichText-yekvnw-0 hIhKZq"
              data-block={true}
              data-editor="text"
              data-offset-key="h1-0-0"
            >
              <div
                className="public-DraftStyleDefault-block public-DraftStyleDefault-ltr"
                data-offset-key="h1-0-0"
              >
                <span
                  data-offset-key="h1-0-0"
                  style={Object {}}
                >
                  <span
                    data-text="true"
                  >
                    Header one
                  </span>
                </span>
              </div>
            </h1>
            <h2
              className="RichText-yekvnw-0 hIhKZq"
              data-block={true}
              data-editor="text"
              data-offset-key="h2-0-0"
            >
              <div
                className="public-DraftStyleDefault-block public-DraftStyleDefault-ltr"
                data-offset-key="h2-0-0"
              >
                <span
                  data-offset-key="h2-0-0"
                  style={Object {}}
                >
                  <span
                    data-text="true"
                  >
                    Header two
                  </span>
                </span>
              </div>
            </h2>
            <h3
              className="RichText-yekvnw-0 hIhKZq"
              data-block={true}
              data-editor="text"
              data-offset-key="h3-0-0"
            >
              <div
                className="public-DraftStyleDefault-block public-DraftStyleDefault-ltr"
                data-offset-key="h3-0-0"
              >
                <span
                  data-offset-key="h3-0-0"
                  style={Object {}}
                >
                  <span
                    data-text="true"
                  >
                    Header three
                  </span>
                </span>
              </div>
            </h3>
            <h4
              className="RichText-yekvnw-0 hIhKZq"
              data-block={true}
              data-editor="text"
              data-offset-key="h4-0-0"
            >
              <div
                className="public-DraftStyleDefault-block public-DraftStyleDefault-ltr"
                data-offset-key="h4-0-0"
              >
                <span
                  data-offset-key="h4-0-0"
                  style={Object {}}
                >
                  <span
                    data-text="true"
                  >
                    Header four
                  </span>
                </span>
              </div>
            </h4>
            <h5
              className="RichText-yekvnw-0 hIhKZq"
              data-block={true}
              data-editor="text"
              data-offset-key="h5-0-0"
            >
              <div
                className="public-DraftStyleDefault-block public-DraftStyleDefault-ltr"
                data-offset-key="h5-0-0"
              >
                <span
                  data-offset-key="h5-0-0"
                  style={Object {}}
                >
                  <span
                    data-text="true"
                  >
                    Header five
                  </span>
                </span>
              </div>
            </h5>
            <h6
              className="RichText-yekvnw-0 hIhKZq"
              data-block={true}
              data-editor="text"
              data-offset-key="h6-0-0"
            >
              <div
                className="public-DraftStyleDefault-block public-DraftStyleDefault-ltr"
                data-offset-key="h6-0-0"
              >
                <span
                  data-offset-key="h6-0-0"
                  style={Object {}}
                >
                  <span
                    data-text="true"
                  >
                    Header six
                  </span>
                </span>
              </div>
            </h6>
            <div
              className="RichText-yekvnw-0 hIhKZq"
              data-block={true}
              data-editor="text"
              data-offset-key="unstyled-0-0"
            >
              <div
                className="public-DraftStyleDefault-block public-DraftStyleDefault-ltr"
                data-offset-key="unstyled-0-0"
              >
                <span
                  data-offset-key="unstyled-0-0"
                  style={Object {}}
                >
                  <span
                    data-text="true"
                  >
                    This block is draft.js content.
                  </span>
                </span>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots Content Page 1`] = `null`;

exports[`Storyshots Content Table 1`] = `
<div
  className="Item-sc-10s0ojn-0 Table__Container-u530n6-0 lNMTM"
  id="table"
  type="table"
>
  <table
    className="Table__StyledTable-u530n6-1 hvaebX"
    scale={1}
  >
    <thead>
      <tr>
        <th
          className="TableCell-oc851c-0 lnUsfb"
        >
          first column
        </th>
        <th
          className="TableCell-oc851c-0 lnUsfb"
        >
          second column
        </th>
        <th
          className="TableCell-oc851c-0 lnUsfb"
        >
          third column
        </th>
      </tr>
    </thead>
    <tbody>
      <tr>
        <td
          className="TableCell-oc851c-0 lnUsfb"
        >
          row 1, column 1
        </td>
        <td
          className="TableCell-oc851c-0 lnUsfb"
        >
          row 1, column 2
        </td>
        <td
          className="TableCell-oc851c-0 lnUsfb"
        >
          row 1, column 3
        </td>
      </tr>
      <tr>
        <td
          className="TableCell-oc851c-0 lnUsfb"
        >
          row 2, column 1
        </td>
        <td
          className="TableCell-oc851c-0 lnUsfb"
        >
          row 2, column 2
        </td>
        <td
          className="TableCell-oc851c-0 lnUsfb"
        >
          row 2, column 3
        </td>
      </tr>
      <tr>
        <td
          className="TableCell-oc851c-0 lnUsfb"
        >
          row 3, column 1
        </td>
        <td
          className="TableCell-oc851c-0 lnUsfb"
        >
          row 3, column 2
        </td>
        <td
          className="TableCell-oc851c-0 lnUsfb"
        >
          row 3, column 3
        </td>
      </tr>
    </tbody>
  </table>
</div>
`;

exports[`Storyshots Content Text 1`] = `
<div
  className="Item-sc-10s0ojn-0 Xwmnj RichText-yekvnw-0 hIhKZq DraftEditor/root DraftEditor/alignLeft RichText-yekvnw-0 hIhKZq DraftEditor/root DraftEditor/alignLeft"
  id="text"
  type="draft"
>
  <div
    className="DraftEditor-root"
  >
    <div
      className="DraftEditor-editorContainer"
    >
      <div
        aria-activedescendant={null}
        aria-autocomplete={null}
        aria-controls={null}
        aria-expanded={null}
        aria-owns={null}
        className="public-DraftEditor-content"
        contentEditable={false}
        onBeforeInput={[Function]}
        onBlur={[Function]}
        onCompositionEnd={[Function]}
        onCompositionStart={[Function]}
        onCopy={[Function]}
        onCut={[Function]}
        onDragEnd={[Function]}
        onDragEnter={[Function]}
        onDragLeave={[Function]}
        onDragOver={[Function]}
        onDragStart={[Function]}
        onDrop={[Function]}
        onFocus={[Function]}
        onInput={[Function]}
        onKeyDown={[Function]}
        onKeyPress={[Function]}
        onKeyUp={[Function]}
        onMouseUp={[Function]}
        onPaste={[Function]}
        onSelect={[Function]}
        role={null}
        spellCheck={false}
        style={
          Object {
            "WebkitUserSelect": "text",
            "outline": "none",
            "userSelect": "text",
            "whiteSpace": "pre-wrap",
            "wordWrap": "break-word",
          }
        }
        suppressContentEditableWarning={true}
      >
        <div
          data-contents="true"
        >
          <h1
            className="RichText-yekvnw-0 hIhKZq"
            data-block={true}
            data-editor="text"
            data-offset-key="h1-0-0"
          >
            <div
              className="public-DraftStyleDefault-block public-DraftStyleDefault-ltr"
              data-offset-key="h1-0-0"
            >
              <span
                data-offset-key="h1-0-0"
                style={Object {}}
              >
                <span
                  data-text="true"
                >
                  Header one
                </span>
              </span>
            </div>
          </h1>
          <h2
            className="RichText-yekvnw-0 hIhKZq"
            data-block={true}
            data-editor="text"
            data-offset-key="h2-0-0"
          >
            <div
              className="public-DraftStyleDefault-block public-DraftStyleDefault-ltr"
              data-offset-key="h2-0-0"
            >
              <span
                data-offset-key="h2-0-0"
                style={Object {}}
              >
                <span
                  data-text="true"
                >
                  Header two
                </span>
              </span>
            </div>
          </h2>
          <h3
            className="RichText-yekvnw-0 hIhKZq"
            data-block={true}
            data-editor="text"
            data-offset-key="h3-0-0"
          >
            <div
              className="public-DraftStyleDefault-block public-DraftStyleDefault-ltr"
              data-offset-key="h3-0-0"
            >
              <span
                data-offset-key="h3-0-0"
                style={Object {}}
              >
                <span
                  data-text="true"
                >
                  Header three
                </span>
              </span>
            </div>
          </h3>
          <h4
            className="RichText-yekvnw-0 hIhKZq"
            data-block={true}
            data-editor="text"
            data-offset-key="h4-0-0"
          >
            <div
              className="public-DraftStyleDefault-block public-DraftStyleDefault-ltr"
              data-offset-key="h4-0-0"
            >
              <span
                data-offset-key="h4-0-0"
                style={Object {}}
              >
                <span
                  data-text="true"
                >
                  Header four
                </span>
              </span>
            </div>
          </h4>
          <h5
            className="RichText-yekvnw-0 hIhKZq"
            data-block={true}
            data-editor="text"
            data-offset-key="h5-0-0"
          >
            <div
              className="public-DraftStyleDefault-block public-DraftStyleDefault-ltr"
              data-offset-key="h5-0-0"
            >
              <span
                data-offset-key="h5-0-0"
                style={Object {}}
              >
                <span
                  data-text="true"
                >
                  Header five
                </span>
              </span>
            </div>
          </h5>
          <h6
            className="RichText-yekvnw-0 hIhKZq"
            data-block={true}
            data-editor="text"
            data-offset-key="h6-0-0"
          >
            <div
              className="public-DraftStyleDefault-block public-DraftStyleDefault-ltr"
              data-offset-key="h6-0-0"
            >
              <span
                data-offset-key="h6-0-0"
                style={Object {}}
              >
                <span
                  data-text="true"
                >
                  Header six
                </span>
              </span>
            </div>
          </h6>
          <div
            className="RichText-yekvnw-0 hIhKZq"
            data-block={true}
            data-editor="text"
            data-offset-key="unstyled-0-0"
          >
            <div
              className="public-DraftStyleDefault-block public-DraftStyleDefault-ltr"
              data-offset-key="unstyled-0-0"
            >
              <span
                data-offset-key="unstyled-0-0"
                style={Object {}}
              >
                <span
                  data-text="true"
                >
                  This block is draft.js content.
                </span>
              </span>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
